package.json

```diff
@@ -4,14 +4,14 @@
   "author": "Workday, Inc. (https://www.workday.com)",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@babel/core": "^7.2.2",
-    "@babel/plugin-proposal-class-properties": "^7.3.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.3.2",
-    "@babel/plugin-transform-modules-commonjs": "^7.2.0",
-    "@babel/plugin-transform-runtime": "^7.4.4",
-    "@babel/preset-env": "^7.3.1",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/preset-typescript": "^7.1.0",
+    "@babel/core": "^7.9.0",
```

Comment:
Note: These babel upgrades require me to throw out my `node_modules` every time I go back to an old branch. May want to consider holding off on this for now.